app/javascript/lib/composables/use_fuzzy_typeahead.ts

```diff
@@ -27,7 +27,8 @@ export function useFuzzyTypeahead<T extends object>({
     if (query.value === '') {
       return searchables;
     } else {
-      return map(fuse.value.search(query.value), 'item');
+      const searchResult = fuse.value.search(query.value);
```

Comment:
Extracting this variable makes it easier to log it for debugging purposes, and I think that breaking it out and capturing it as a named variable also might help with readability, or at least probably doesn't hurt it.